src/mango/test/07-text-custom-field-list-test.py

```diff
@@ -160,3 +161,52 @@ def test_all_match(self):
         })
         assert len(docs) == 1
         assert docs[0]["user_id"] == 10
+
+@unittest.skipUnless(mango.has_text_service(), "requires text service")
+class CustomFieldsExistsTest(mango.UserDocsTextTests):
+
+    FIELDS = [
+        {"name": "exists_field", "type": "string"},
+        {"name": "exists_array.[]", "type": "string"},
+        {"name": "exists_object.should", "type": "string"},
+        {"name": "twitter", "type": "string"}
+    ]
+
+    def test_exists_field(self):
+        docs = self.db.find({"exists_field": {"$exists": True}})
+        assert len(docs) == 2
```

Comment:
can you use the `self.assertEqual` function instead (e.g. https://github.com/apache/couchdb/pull/971/files#diff-ce3549b9dcbffaa7483d1f289451c786R453) as this provides much better error messages when assertions fail.